tests/core/test_actor.py

```diff
@@ -15,15 +15,49 @@
 from tests import dummy_mixer
 
 
+def make_backend_mock(
+    actor_classname,
+    uri_schemes,
+    has_library,
+    has_library_browse,
+    has_playback,
+    has_playlists,
+):
```

Comment:
I think we should force keyword arguments here. It makes the tests a bit more verbose, but also more readable.

```suggestion
def make_backend_mock(
    *,
    actor_classname,
    uri_schemes,
    has_library,
    has_library_browse,
    has_playback,
    has_playlists,
):
```

Reply:
Sure, I think that's a good idea. I think `actor_classname` should remain as a compulsory positional parameter though, because it's always likely to be short and straightforward.